ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json

```diff
@@ -22,5 +22,19 @@
     "Label": "Is GA for LIP",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "ID": "claimantUserDetails",
+    "Label": " ",
+    "FieldType": "GAUserDetailsGAspec",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "ID": "defendantUserDetails",
+    "Label": " ",
+    "FieldType": "GAUserDetailsGAspec",
+    "SecurityClassification": "Public"
```

Comment:
did we miss "Searchable": "N" as it's new field?

Reply:
> did we miss "Searchable": "N" as it's new field?

added